src/pages/home-page-slides/slide-details/slide-details.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { useStyles } from './slide-details.styles';
+import HomePageSlideForm from '../../../components/home-page-slide-form';
+import { getAvailableSlides, getSlide } from '../../../redux/home-page-slides/home-page-slides.actions';
+import LoadingBar from '../../../components/loading-bar';
+
+const SlideDetails = ({ match }) =>{
+  const { id } = match.params;
+  const styles = useStyles();
+  const dispatch = useDispatch();
+  const { loading, slide,availableSlides } = useSelector(({ Slides }) => ({
+    loading: Slides.slideLoading,
+    slide: Slides.slide,
+    availableSlides: Slides.availableSlides
+  }));
+
+  useEffect(() => {
+    dispatch(getSlide(id));
+    dispatch(getAvailableSlides())
+  }, [dispatch]);
+
+  const slideOrder = availableSlides.filter(slideItem=>slideItem.show===true).length + 1
```

Comment:
Add a proper aligning for the code, also you can simplify this filter callback here: filter(slideItem => slideItem.show)

Reply:
Fixed